include/bluetooth/mcc.h

```diff
@@ -26,271 +34,337 @@ extern "C" {
- * @param name          Player name, max length is PLAYER_NAME_MAX (mpl.h)
+ * @param name          Player name
  */
-typedef void (*bt_mcc_player_name_read_cb)(struct bt_conn *conn, int err,
-					   const char *name);
+typedef void (*bt_mcc_player_name_read_cb)(struct bt_conn *conn, int err, const char *name);
 
 #ifdef CONFIG_BT_OTC
-/** @brief Callback function for bt_mcc_read_icon_obj_id()
+/**
+ * @brief Callback function for bt_mcc_read_icon_obj_id()
+ *
+ * Called when the icon object ID is read
  *
  * @param conn          The connection that was used to initialise the media control client
  * @param err           Error value. 0 on success, GATT error or errno on fail
  * @param icon_id       The ID of the Icon Object. This is a UINT48 in a uint64_t
  */
-typedef void (*bt_mcc_icon_obj_id_read_cb)(struct bt_conn *conn, int err,
-					   uint64_t id);
+typedef void (*bt_mcc_icon_obj_id_read_cb)(struct bt_conn *conn, int err, uint64_t id);
 #endif /* CONFIG_BT_OTC */
 
-/** @brief Callback function for bt_mcc_read_icon_url()
+/**
+ * @brief Callback function for bt_mcc_read_icon_url()
+ *
+ * Called when the icon URL is read
  *
  * @param conn          The connection that was used to initialise the media control client
  * @param err           Error value. 0 on success, GATT error or errno on fail
  * @param icon_url      The URL of the Icon
  */
-typedef void (*bt_mcc_icon_url_read_cb)(struct bt_conn *conn, int err,
-					const char *icon_url);
+typedef void (*bt_mcc_icon_url_read_cb)(struct bt_conn *conn, int err, const char *icon_url);
 
-/** @brief Callback function for track changed notifications
+/**
+ * @brief Callback function for track changed notifications
+ *
+ * Called when a track change is notified.
  *
- * The track changed characteristic is a special case.  It can not be
- * read or written, it can only be notified.
+ * The track changed characteristic is a special case.  It can not be read or
+ * set, it can only be notified.
  *
  * @param conn          The connection that was used to initialise the media control client
  * @param err           Error value. 0 on success, GATT error or errno on fail
  */
 typedef void (*bt_mcc_track_changed_ntf_cb)(struct bt_conn *conn, int err);
 
 
-/** @brief Callback function for bt_mcc_read_track_title()
+/**
+ * @brief Callback function for bt_mcc_read_track_title()
+ *
+ * Called when the track title is read or notified
  *
  * @param conn          The connection that was used to initialise the media control client
  * @param err           Error value. 0 on success, GATT error or errno on fail
  * @param title         The title of the track
  */
-typedef void (*bt_mcc_track_title_read_cb)(struct bt_conn *conn, int err,
-					   const char *title);
+typedef void (*bt_mcc_track_title_read_cb)(struct bt_conn *conn, int err, const char *title);
 
-/** @brief Callback function for bt_mcc_read_track_dur()
+/**
+ * brief Callback function for bt_mcc_read_track_duration()
```

Comment:
Missed one :) (missing @)

Reply:
Thanks!  It snuck back in when I applied the next commit ;-)
Fixed